dev/flipbook.tsx

```diff
@@ -204,6 +208,9 @@ function SideBySideQuestionRenderer({
 function GradableRenderer(props: QuestionRendererProps) {
     const {question, apiOptions} = props;
     const rendererRef = useRef<Renderer>(null);
+    const [score, setScore] = useState<PerseusScore | undefined>();
```

Comment:
```suggestion
    const [score, setScore] = useState<PerseusScore>();
```
Does the same thing with less code.

Reply:
Weird. I thought I tried that and it complained that I wasn't providing a value. It works as you say. Thanks!